src/test/resources/features/integration/example_unitary.feature

```diff
@@ -0,0 +1,34 @@
+Feature: sample karate test script
+  for help, see: https://github.com/intuit/karate/wiki/IDE-Support
+
+  Background:
+    * url PathUrl
+
+    # conexion BD
+    * def config = { username: #(DB_USERNAME), password: #(DB_PASSWORD), url: #(DB_URL), driverClassName: #(DB_DRIVER) }
+    * def DbUtils = Java.type('Support.DbUtils')
+    * def db = new DbUtils(config)
+
+    @user
+    Scenario: get all users and then get the first user by id
+    Given path 'users'
+    When method get
+    Then status 200
+    * print "Test unitary execute OK"
+
+    @userById
+    Scenario: get  users Basic filtering is supported through query parameters.
+    Given path 'posts'
+    And params { userId: 1}
+    When method get
+    Then status 200
+    * print "Test unitary execute OK"
+
+    @userByIdParams @ignore
+    Scenario: get  users Basic filtering is supported through query parameters.
+    * def rut = { userId: '#(userId)'}
+    * path 'posts'
+    * params { userId: userId}
+    * method get
+    * status 200
+    * print "Test unitary execute OK"
```

Comment:
I suggest to change this _Test unitary_ to **_unit test_** (not only here but every line in the project, I think 'test unitary' is just never used really)